content/200-orm/200-prisma-client/100-queries/060-full-text-search.mdx

```diff
@@ -247,34 +247,80 @@ However, if you try to search on `title` alone, the search will fail with the er
 
 ## Full-text search with raw SQL
 
-Full-text search is currently in Preview and due to a [known issue](https://github.com/prisma/prisma/issues/23627), you may be seeing slow search queries. If that's the case, you can optimize your query using [raw SQL](/orm/prisma-client/queries/raw-database-access).
+Full-text search is currently in Preview and due to a [known issue](https://github.com/prisma/prisma/issues/23627), you may be seeing slow search queries. If that's the case, you can optimize your query using [TypedSQL](/orm/prisma-client/using-raw-sql).
 
 ### PostgreSQL
 
-In PostgreSQL, you can use `to_tsvector` and `to_tsquery` to express your search query:
+With [TypedSQL](/orm/prisma-client/using-raw-sql), you can use PostgreSQL's `to_tsvector` and `to_tsquery` to express your search query.
 
-```ts
-const term = `cat`
-const result =
-  await prisma.$queryRaw`SELECT * FROM "Blog" WHERE to_tsvector('english', "Blog"."content") @@ to_tsquery('english', ${term});`
-```
+<TabbedContent>
+
+  <TabItem value="fullTextSearch.sql">
+
+  ```sql
+  SELECT * FROM "Blog" WHERE to_tsvector('english', "Blog"."content") @@ to_tsquery('english', ${term});
+  ```
+
+  </TabItem>
+
+  <TabItem value="index.ts">
+
+  ```ts
+  const term = `cat`
+  const result = await prisma.$queryRawTyped(fullTextSearch(term))
+  ```
```

Comment:
@jharrell is it worth showing the import here?

```suggestion
  ```ts
  import { fullTextSearch } from "@prisma/client/sql"
  
  const term = `cat`
  const result = await prisma.$queryRawTyped(fullTextSearch(term))
  ```
```